### PR TITLE
Added New Input System

### PR DESCRIPTION
**Pull request by Apearson75**
Added a setting in the preferences that allows you to toggle new input.

It works just like in other mods. It prevents you from losing health by spamming.

image

### CODE REVIEW

File: source/ui/PreferencesMenu.hx [hunk 1]
```diff
@@ -79,7 +81,9 @@ class PreferencesMenu extends ui.OptionsState.Page
 
         if (getPref("fps-counter"))
             FlxG.stage.removeChild(Main.fpsCounter);
-
+        if (getPref("fullscreen")){
```
Comment (Apearson75) [inline source/ui/PreferencesMenu.hx:84]: oops. I was trying to make a fullscreen toggle and it didn't work. I accidentally left it here

**Comment by DotWith** [comment]
FNF is supposed to be a simple game. I shouldn't have to change options to balance my gameplay. Think about the multiplayer how would that work. Later I'm going to make pull-request rules.

**Comment by Cheemsandfriends** [comment]
I know the intentions aren't bad but...
Why? Literally Dot doesn't like, hates and/or has a cult that makes Kadedev as the enemy.
Seriously tho, I don't think this would be pretty good for stuff like multiplayer cos basically noobs will use that always. Not to mention, the engine's input was made based on the old input. So adding the KE Input is basically making it 20000000x easier and boring.

I don't hate that you made the change, i think it's unnecessary to add that when literally the engine is trying to use old input.

**Comment by Apearson75** [comment]
Oh I'm sorry.
I never knew that was your intention.
:P

**Pull request closed without merging**